root/programs/C#/Samples/WebApp_sample/ProjectX_sample/Scripts/touryo/common.js

```diff
@@ -603,81 +684,206 @@ function Fx_ShowMessageDialog(url) {
     url = url + "&Time=" + year_now + month_now + date_now + day_now + hour_now + minute_now + second_now;
 
     // マスクする。
-    Fx_MaskOn();
+    Fx_DialogMaskOn();
```

Comment:
@yusukemaegawa 

We use the following method to determine the size.
- Fx_getContentsHeight()
- Fx_getBrowserWidth()

More appropriate implementation may be possible.